UnitySDK/Assets/ML-Agents/Scripts/Academy.cs

```diff
@@ -203,9 +284,9 @@ void InitializeEnvironment()
                         new CommunicatorInitParameters
                         {
                             version = k_ApiVersion,
-                            name = gameObject.name,
+                            name = "AcademySingleton",
```

Comment:
Should we have a name field? Or default this to the current scene?

Reply:
name used to be given to Python at some point. we removed this (the Python environment is now nameless) so we can delete that field I think.

Reply:
It was coming from the Unity GameObject (e.g. "3DBallAcademy"). I'll remove it from the protos in a subsequent PR.

Reply:
https://jira.unity3d.com/browse/MLA-518 for followup